counter/src/main/java/edu/nextstep/camp/counter/CounterViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package edu.nextstep.camp.counter
+
+import androidx.annotation.StringRes
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class CounterViewModel : ViewModel() {
+
+    val eventShowToast = SingleLiveEvent<Int>()
+
+    private val _count = MutableLiveData(0)
+    val count: LiveData<Int> get() = _count
```

Comment:
get()을 쓰지 않고 바로 할당해주어도 무방합니다.
```suggestion
    val count: LiveData<Int> = _count
```